docs/guide/blog/comment-blog.md

```diff
@@ -5,34 +5,45 @@ order: 2
 
 # Add Associated Comments to a Blog Post
 
-In this tutorial, you will create a new message using list to add comments to a blog post. You will be implementing logic for writing comments to the blockchain as well as querying for blog posts with associated comments and deleting them.
+In this tutorial, you create a new message to add comments to a blog post. These tutorial steps 
 
-You can only add comments to a blog post that is no older than 100 blocks. 
+By completing this tutorial, you will learn about:
 
-**Note:** This value has been hard coded to a lower number for rapid testing. You can increase it to a greater number to achieve longer period of time before commenting is disabled.
+* Scaffolding a new `list` with proto functions and keeper functions
+* Adding comments to existing blog posts
+* Querying for blog posts that have associated comments
+* Deleting comments from a blog post
+* Implementing logic for writing comments to the blockchain
 
-### Prerequisites:
+**Note:** For this tutorial, adding comments is available only to blog posts that are no older than 100 blocks. The 100 block value has been hard coded for rapid testing. You can increase the block count to a larger number to achieve a longer time period before commenting is disabled.
 
-- This tutorial is an extension of previously written blog tutorial. Make sure you complete that first before proceeding with this tutorial.
-- This tutorial assumes basic knowledge of blog tutorial implementation.
-- Make sure you are inside the `blog` directory created in the previous blog tutorial.
-- In this chapter, you will use the command - `starport scaffold list comment --no-message` to get all the useful functions. Make sure to familiarize yourself with the command.
+## Prerequisites
 
-By completing this tutorial, you will learn about:
+- This tutorial is an extension of and requires completion of the [Module Basics: Build a Blog](index.md) tutorial. 
 
-* Scaffolding a new `list` with proto functions and keeper functions
-* Add comments to existing blog post
-* Display the blog post by ID with associated comments
-* Delete comments from a given blog post
+## Core Concepts 
+
+This tutorial relies on the `blog` blockchain that you built in the build a blog tutorial. 
+
+To get the useful functions for this tutorial, you use the `starport scaffold list comment --no-message` command. Make sure to familiarize yourself with the command.
 
 
-## Fetch functions using list command
+## Fetch Functions Using List Command
```

Comment:
@alijnmerchant21  be sure to review the tutorial template, edit the file to see all of the stylistic guidelines https://github.com/tendermint/starport/edit/develop/docs/contributing/templates/tutorial-template.md 

For example 

> `<!-- Use Title Case for all Titles, see https://capitalizemytitle.com/ -->`